javascript-conditionals/main.js

```diff
@@ -76,7 +76,7 @@ console.log('isOldEnoughToDrinkandDrive value:', isOldEnoughToDrinkAndDrive());
 function categorizeAcidity(pH) {
   if (pH === 7) {
     return 'neutral';
-  } else if (pH >= 0 && pH <= 7) {
+  } else if (pH <= 0 && pH <= 7) {
```

Comment:
This should be greater than or equal to, not less than or equal to.

Reply:
<img width="369" alt="Screenshot 2023-07-13 at 10 50 16 PM" src="https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/aa927d85-77d2-438e-88f4-349a8f31bd2f">

sorry! I feel dyslexic with greater than or less than sometimes...